Editor/BuildScriptEditor.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using GameWorkstore.Automation;
+using UnityEditor;
+using UnityEngine;
+
+namespace GameworkStore.Automation.Editor
+{
+    [CustomEditor(typeof(BuildScript))]
+    public class BuildScriptEditor : UnityEditor.Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            base.OnInspectorGUI();
+            BuildScript buildScript = (BuildScript)target;
+            if(!buildScript) return;
+            Show(buildScript);
+        }
+
+        public void Show(BuildScript buildScript)
+        {
+            ShowButtonCreate<AndroidBuildPlatform>(buildScript, nameof(AndroidBuildPlatform));
+            ShowButtonCreate<IOSBuildPlatform>(buildScript, nameof(IOSBuildPlatform));
+            ShowButtonCreate<WindowsBuildPlatform>(buildScript, nameof(WindowsBuildPlatform));
+            ShowButtonCreate<LinuxBuildPlatform>(buildScript, nameof(LinuxBuildPlatform));
+            ShowButtonCreate<MacOSBuildPlatform>(buildScript, nameof(MacOSBuildPlatform));
+            ShowButtonCreate<ServerWindowsBuildPlatform>(buildScript, nameof(ServerWindowsBuildPlatform));
+            ShowButtonCreate<ServerMacOSBuildPlatform>(buildScript, nameof(ServerMacOSBuildPlatform));
+            ShowButtonCreate<ServerLinuxBuildPlatform>(buildScript, nameof(ServerLinuxBuildPlatform));
+            ShowButtonCreate<WebGLBuildPlatform>(buildScript, nameof(WebGLBuildPlatform));
+            ShowButtonCreate<UWPBuildPlatform>(buildScript, nameof(UWPBuildPlatform));
+
+        }
+
+        public static void ShowButtonCreate<T>(BuildScript buildScript, string name) where T : BuildPlataform
+        {
+            if(buildScript.Has<T>()) return;
+            // EditorGUI.BeginDisabledGroup(buildScript.Has<T>());
+            if (GUILayout.Button("New "+name+" Plataform"))
+            {
+                MakeNewAndroidBuildPlatform<T>(buildScript, name);
```

Comment:
MakeNewAndroidBuildPlatform sounds like it only works for Android, can change it to something more generic if it's used for all platforms?